Physics/Physics.cpp

```diff
@@ -0,0 +1,72 @@
+#include "Physics.h"
+
+float Physics::CalculateLinearForce(float mass, float acceleration)
+{
+	return mass * acceleration;
+}
+
+float Physics::CalculateLinearMomentum(float mass, float fvelocity)
+{
+	return mass * fvelocity;
+}
+
+float Physics::CalculateLinearKineticEnergy(float mass, float fvelocity)
+{
+	return (mass * (fvelocity * fvelocity)) / 2.0f;
+}
+
+float Physics::CalculateLinearDisplacement(float ivelocity, float fvelocity, float time)
+{
+	return (ivelocity + fvelocity) * time / 2.0f;
+}
+
+float Physics::CalculateLinearDisplacement(float ivelocity, float acceleration, float time)
+{
+	return (ivelocity * time) + (acceleration * (time * time) / 2.0f);
+}
```

Comment:
These two functions clash: return the same, named the same with the same number and type of arguments. Need to rename one of them